packages/cli/src/commands/mock.ts

```diff
@@ -57,15 +58,23 @@ const mockCommand: CommandModule = {
         },
       }),
   handler: parsedArgs => {
-    const { multiprocess, dynamic, port, host, cors, operations } = (parsedArgs as unknown) as CreatePrismOptions & {
+    const {
+      multiprocess,
+      dynamic,
+      port,
+      host,
+      cors,
+      operations,
+      spec,
+    } = (parsedArgs as unknown) as CreatePrismOptions & {
       multiprocess: boolean;
+      spec: string;
     };
 
-    if (multiprocess) {
-      return createMultiProcessPrism({ cors, dynamic, port, host, operations });
-    }
+    const createPrism = multiprocess ? createMultiProcessPrism : createSingleProcessPrism;
+    const options = { cors, dynamic, port, host, operations };
 
-    return createSingleProcessPrism({ cors, dynamic, port, host, operations });
+    return runPrism(createPrism, options, spec);
```

Comment:
```suggestion
    return runPrismAndSetupWatcher(createPrism, options, spec);
```

or anything that gives the idea of what's going on.